EIPS/eip-4675.md

```diff
@@ -0,0 +1,212 @@
+---
+eip: 4675
+title: Multi-Fractional Non-Fungible Token Standard
+description: Token standard to handle multiple NFTs in a fractionalized form
```

Comment:
```suggestion
title: Multi-Fractional Non-Fungible Tokens
description: Fractionalize multiple NFTs using a single contract
```

We try to avoid "standard" in titles and descriptions.